x/projects/keeper/creation.go

```diff
@@ -260,9 +261,16 @@ func (k Keeper) unregisterKey(ctx sdk.Context, key types.ProjectKey, project *ty
 		if err != nil {
 			return err
 		}
+
 		found = project.DeleteKey(types.ProjectDeveloperKey(key.Key))
 		if !found {
-			panic("unregisterKey: developer key not found")
+			// the developer key must be found, because it was already found in
+			// the developerKeysFS for this project; return error to avoid panic.
+			return utils.LavaFormatWarning("critical: develper key missing in project", sdkerrors.ErrNotFound,
```

Comment:
is this supposed to be LavaFormatWarning (can a warning be critical)?
also develper -> developer

Reply:
nice catch. will fix.